features/formations.feature

```diff
@@ -0,0 +1,79 @@
+Feature: formations
+  As an admin I am able to CRUD a Formation
+  
+  @createSchema
+  Scenario: I can see Formations Homepage
+    Given I am logged in as "admin"
+    Given I am on "/formations"
+    Then the response status code should be 200
+    Then I should see "Liste des formations"
+  
+  Scenario: I can see Formations Homepage
+    Given I am logged in as "admin"
+    Given I am on "/formations/admin"
+    Then the response status code should be 200
+    Then I should see "Formations - Espace Admin"
+  
+  Scenario: I can see a Formation
+    Given I am logged in as "admin"
+    Given I am on "/formations/3"
+    Then the response status code should be 200
+    Then I should see "Modifier une formation"
+    And I should see "Télécharger les documents"
+    and I should see "Supprimer la formation"
```

Comment:
`and` n'est pas reconnu comme mot clé Behat. Pour que le test fonctionne, il faut utiliser `And`.

La même chose pour plusieurs pas dans le document.